skyportal/models/localization.py

```diff
@@ -219,6 +230,35 @@ def center(self):
 
         return center_info
 
+    def get_localization_path(self):
+        """
+        Get the path to the localization's data.
+        """
+        return self._localization_path
+
+    def save_data(self, filename, file_data):
+        """
+        Save the localization's data to disk.
+        """
+
+        # there's a default value but it is best to provide a full path in the config
+        root_folder = cfg.get('localizations_folder', 'localizations_data')
+
+        full_path = save_file_data(root_folder, str(self.id), filename, file_data)
+
+        # persist the filename
+        self._localization_path = full_path
+
+    def delete_data(self):
+        """
+        Delete the localizations's data from disk.
+        """
+
+        delete_file_data(self._attachment_path)
+
+        # reset the filename
+        self._localization_path = None
```

Comment:
Is there a better way to handle errors in this part? For example, if the file is not found, and we remove the path to it, we lose our ability to track it down later. I'm not sure what the right thing to do here.

Reply:
My first instinct would be to let the delete_file_data pass on any exceptions it has, and catch it outside, where you can put the `self._localization_path = None` line inside the try/except block. That could mean we leave path strings that are dead ends, but you'd have the opportunity to re-check and delete them explicitly. 
But that's just one option, and we should consider what is the best policy.

Reply:
done